modules/service/rest-api-v2.js

```diff
@@ -601,6 +606,52 @@ class RestAPIServiceV2 {
         });
     }
 
+    async _getChallenges(req, res) {
+        if (req.body === undefined) {
+            res.status(400);
+            res.send({
+                message: 'Bad request',
+            });
+            return;
+        }
+
+        // Check if import type is valid
+        if (req.body.startDate === undefined ||
+            req.body.endDate === undefined) {
+            res.status(400);
+            res.send({
+                message: 'Bad request startDate and endDate required!',
+            });
+            return;
+        }
+
+        const challenges = await Models.challenges.findAll({
+            where: {
+                start_time: {
```

Comment:
Let's prevent fetching challenges that weren't sent yet, we can probably filter those by the status or something